dev/manual_tests/lib/overlay_geometry.dart

```diff
@@ -133,7 +133,7 @@ class CardBuilder extends SliverChildDelegate {
 }
 
 class OverlayGeometryAppState extends State<OverlayGeometryApp> {
-  List<CardModel> cardModels;
+  List<CardModel>? cardModels;
```

Comment:
Nullable collection properties can be avoided
```suggestion
  List<CardModel> cardModels = <CardModel>[];
```